aiosmtpd/handlers.py

```diff
@@ -138,7 +137,7 @@ def from_cli(cls, parser, *args):
             parser.error('Sink handler does not accept arguments')
         return cls()
 
-    def process_message(self, peer, mailfrom, rcpttos, data, **kws):
+    def process_message(self, session, envelope):
```

Comment:
This one shouldn't change its argument list, or it should be changed to `handle_DATA()` and made a coroutine.